frontend/pages/queries/ManageQueriesPage/ManageQueriesPage.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useCallback, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { push } from "react-router-redux";
+import { IQuery } from "interfaces/query";
+import { IUser } from "interfaces/user";
+
+// @ts-ignore
+import queryActions from "redux/nodes/entities/queries/actions";
+// @ts-ignore
+import { renderFlash } from "redux/nodes/notifications/actions";
+
+import paths from "router/paths";
+import permissionUtils from "utilities/permissions";
+
+import Button from "components/buttons/Button";
+import QueriesListError from "./components/QueriesListError";
+import QueriesListWrapper from "./components/QueriesListWrapper";
+import RemoveQueryModal from "./components/RemoveQueryModal";
+
+const baseClass = "manage-queries-page";
+interface IRootState {
+  auth: {
+    user: IUser;
+  };
+  entities: {
+    queries: {
+      isLoading: boolean;
+      data: IQuery[];
+      errors: any;
+    };
+  };
+}
+
+const renderTable = (
+  onRemoveQueryClick: React.MouseEventHandler<HTMLButtonElement>,
+  queriesList: IQuery[],
+  queriesErrors: any
+): JSX.Element => {
+  if (Object.keys(queriesErrors).length !== 0) {
```

Comment:
Let's use `length > 0` here. `!==` implies that a negative number can render the result.